css/style.css

```diff
@@ -7,22 +7,23 @@
   --accent-green: #cbefb6;
   --theme-secondary: #0f362e;
   --theme-soft-gray: rgba(0, 0, 0, 0.05);
-  --spacing-1: 6px;
-  --spacing-2: 9px;
-  --spacing-3: 12px;
-  --spacing-4: 18px;
+  --spacing-1: 4px;
+  --spacing-2: 6px;
+  --spacing-3: 8px;
+  --spacing-4: 10px;
   --spacing-5: 24px;
   --spacing-6: 30px;
-  --spacing-7: 36px;
-  --spacing-8: 42px;
+  --spacing-7: 32px;
+  --spacing-8: 36px;
+  --spacing-9: 42px;
   --rounded: 8px;
-  --font-xxl: 3.5rem;
+  --font-xxl: 4.875rem;
```

Comment:
@Right-Noe are you sure about that? 4.875rem makes it 78px. It should be checked in all places where this variable is used: .section--headline h1

Reply:
You're right, @martinosmartinos . I'll change the font size just for that section title (section-inspiration) and review all the headers font size.